src/components/Loading/Loading.tsx

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled, { css } from 'styled-components';
+
+import { Padbox, Stack } from '../layout';
+import { ProgressBar } from '../ProgressBar';
+import { Spinner } from '../Spinner';
+import { Text } from '../typographyLegacy';
+import { pxToRem } from '../../utils';
+import { LoadingRootProps } from './Loading.types';
+
+const LoadingRoot = styled(Padbox)<LoadingRootProps>`
+  display: flex;
+  align-items: center;
+  ${({ $height }) => css`
+    min-height: ${pxToRem($height)};
+  `};
```

Comment:
```suggestion
  min-height: ${({ $height }) => pxToRem($height)};
```
I don't think we need to bring `css` to this.